packages/webpack-loader/tests/parser.test.ts

```diff
@@ -31,10 +31,10 @@ test('fragment spread', () => {
   const docStr = /* GraphQL */`
     query Foo {
       foo {
-        ...fooFrgmnt
+        ...fooFragment
```

Comment:
Yes, `foo` is a word as far as people are concerned, but in tests, there's really no need to drop vowels.